packages/preset-tailwind/src/index.ts

```diff
@@ -237,7 +238,7 @@ export const buttons = {
   },
 }
 
-export const colors = {
+export const colors: Theme['colors'] = {
```

Comment:
Do we have to annotate `colors` and `baseFonts`?
I'm thinking about exact theme types, and annotating it explicitly here would forget information about color names / font family names.